app/src/main/java/org/schabi/newpipe/fragments/list/playlist/PlaylistFragment.java

```diff
@@ -226,10 +228,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 NavigationHelper.openSettings(requireContext());
                 break;
             case R.id.menu_item_openInBrowser:
-                ShareUtils.openUrlInBrowser(requireContext(), url);
+                ShareUtils.openUrlInBrowser(requireContext(), cleanUrl == null ? url : cleanUrl);
                 break;
             case R.id.menu_item_share:
-                ShareUtils.shareUrl(requireContext(), name, url);
+                ShareUtils.shareUrl(requireContext(), name, cleanUrl == null ? url : cleanUrl);
```

Comment:
You could just use `currentInfo.getUrl()` here as well

Reply:
So I should replace `cleanUrl == null ? url : cleanUrl` with `currentInfo.getUrl()`? I can basically revert the last commit and make this change instead, right?